content/en/docs/examples/alb-nlb/index.md

```diff
@@ -0,0 +1,363 @@
+---
+title: Securely direct traffic from IBM Cloud Kubernetes Service Ingress to Istio Ingress Gateway
+description: Configuring the IKS Ingress ALB to direct traffic Istio Ingress gateway with mutual TLS.
+weight: 43
+keywords: [traffic-management,ingress,file-mount-credentials,iks]
+---
+
+This example shows how to configure the [IBM Cloud Kubernetes Service](https://www.ibm.com/cloud/kubernetes-service/)
+[Ingress ALB](https://cloud.ibm.com/docs/containers?topic=containers-ingress-about) to direct traffic to the Istio
+ingress gateway with [NLB](https://cloud.ibm.com/docs/containers?topic=containers-loadbalancer-about), while securing
+the traffic between them using {{< gloss  >}}mutual TLS authentication{{< /gloss >}}.
+
+When you use IBM Cloud Kubernetes Service without Istio, you may control your ingress traffic using the provided
+Application Load Balancer (ALB). This ingress-traffic routing is configured using a Kubernetes
+[Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/) resource with
+[ALB-specific annotations](https://cloud.ibm.com/docs/containers?topic=containers-ingress_annotation).
+
+When you start using Istio in your IKS cluster, it is recommended that you use the
+[Istio Ingress Gateway](/docs/tasks/traffic-management/ingress/ingress-control/) instead of using Kubernetes Ingress
+resource. One of the main reasons to use Istio ingress gateway is the fact the ALB provided by IKS will not be able to
+communicate with the services inside the mesh when you enable Istio mutual TLS. While you transition to having only
+Istio ingress gateway as your main entry point, you can continue to use the traditional Ingress for non-Istio services
+and the Istio ingress gateway for services that are part of the mesh.
+
+In this example you will configure the IKS Ingress ALB to direct traffic to the services inside an Istio service mesh
+through the Istio ingress gateway, while using mutual TLS between ALB and the gateway.
+The traffic to the services without Istio sidecar can continue to flow as before, directly from ALB.
```

Comment:
I think a diagram can help here. Maybe something like this?
<img width="591" alt="image" src="https://user-images.githubusercontent.com/5502967/66315731-5b180880-e8e4-11e9-886e-763356b616ab.png">

Reply:
yep, let me create a diagram

Reply:
I think this diagram isn't entirely correct.  it would be 2 flows:
1. Client - ALB - istio ingress gateway - pod
2. Client - NLB - istio ingress gateway - pod

Agree?

Reply:
Right, the diagram needs to be modified a bit to show both flows

Reply:
right, I will change the diagram according to the example